### PR TITLE
4.4 Solo Challenge

### PR DESCRIPTION
**Pull request by lrrnlds**
@Bontgoy Hi Tyler, I'd love to have my code reviewed if possible! Thanks!

### CODE REVIEW

**Comment by Bontgoy** [comment]
If requesting feedback make sure to wait for my reply before merging the pull request

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		if allergies == "sunshine"
+			break
+
+			puts result
+
+		end
+	end
+
+#create if statements for age, garlic bread and insurance
+	if 2016-birth_year == (age)
```
Comment (Bontgoy) [inline ruby/vampires.rb]: with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	if age == true && (garlic_bread == true || insurance == true)
+		result = "Probably not a vampire"
+	end
+	if age == false && (garlic_bread == false || insurance == false)
+		result = "Probably a vampire"
+	end
+	if age == false && garlic_bread == false && insurance == false
+		result = "Almost certainly a vampire"
+	end
+	if name == ("Drake Cula" || "Tu Fang")
```
Comment (Bontgoy) [inline ruby/vampires.rb]: I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.
```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```

**Comment by lrrnlds** [comment]
Thanks so much Tyler! I'll definitely try out your comments to my code

**Comment by Bontgoy** [comment]
Awesome!  Keep me posted on your progress.

**Pull request merged**